CONTRIBUTING.md

```diff
@@ -46,6 +46,32 @@ development of the library, from your local clone of the fork, run
 (myenv) pip install -r dev_requirements.txt
 ```
 
+#### Special Note for Windows Users Using Python 3.8:
+To prevent errors when running `make docs` and `make doctest`, Windows developers using Pythong 3.8 will also need to edit \_\_init\_\_.py in their environment's asyncio directory.
+This is due to asyncio changing their default event loop beginning in Python 3.8, and the default event loop will not support Unix-style APIs used by some dependencies.
+1. Locate your environment directory (likely C:\Users\{username}\anaconda3\envs\\{your_env}), and open {env_dir}/Lib/asyncio/\_\_init\_\_.py.
+2. Add `import asyncio` to the file's import statements.
+3. Find the block of code below and replace it with the provided replacement.
+    * Original Code  
+
+          if sys.platform == 'win32':  # pragma: no cover
+              from .windows_events import *
+              __all__ += windows_events.__all__
+          else:
+              from .unix_events import *  # pragma: no cover
+              __all__ += unix_events.__all__
+  
+    * Replacement Code  
+
+          if sys.platform == 'win32':  # pragma: no cover
+              from .windows_events import *
+              asyncio.set_event_loop_policy(asyncio.WindowsSelectorEventLoopPolicy())
+              __all__ += windows_events.__all__
+          else:
+              from .unix_events import *  # pragma: no cover
```

Comment:
This could be done the way we format code in markdown.

Reply:
This is formatted such that is appears as a nested code block instead of a full-width code block. But if you would like it full-width instead of nested, I can change it.
![Screenshot 2021-05-24 160632](https://user-images.githubusercontent.com/20098360/119359544-0a684580-bcaa-11eb-93cc-947964600f95.png)